### ISSUE TITLE
not sure if 4.2 is installing?

### ISSUE THREAD

**Issue by BarryAllen2024** [enchanted-sword/dashboard-unfucker#92]
**Describe the bug**
Honestly, not sure if this is a github/tampermonkey issue or a code issue. When I open tampermonkey to see the running scripts, I see version 4.2, but on Tumblr it says version 4.1.7b. Morever, the issue with the loading grey boxes being avatars hasn't changed, so I'm thinking I'm not getting the 4.2 update. I tried installing in a private browser (no adblockers) and the same thing happens, so I don't think it's those.

**To Reproduce**
Basically what I said above.

**Version (please complete the following information):**
 - Browser and version: Firefox 116.0.3
 - Version: 4.1.7b?

**Comment by enchanted-sword** [comment]
Whoops, should be good now
